docs/src/extend/custom-rules.md

```diff
@@ -514,7 +515,7 @@ The `SourceCode` object is the main object for getting more information about th
 ```js
 module.exports = {
     create: function(context) {
-        var sourceCode = context.getSourceCode();
+        var sourceCode = context.sourceCode; // or use "context.getSourceCode()", but that is deprecated.
```

Comment:
```suggestion
        var sourceCode = context.sourceCode;
```

I think there's no need to mention the deprecated method in the example, but we could add a deprecation note paragraph like in [Accessing Variable Scopes](https://eslint.org/docs/latest/extend/custom-rules#accessing-variable-scopes).

We should also update line 513: `` by using the `context.getSourceCode()` method `` -> `` by using the `context.sourceCode` property ``

Reply:
@mdjermanovic updated 👍🏻